src/main/java/dev/breeze/settlements/entities/cats/sensors/CatSensorType.java

```diff
@@ -4,6 +4,11 @@
 
 public class CatSensorType {
 
+    /**
+     * Sensor for detecting owner
+     */
+    public static final String REGISTRY_KEY_OWNER = "settlements_cat_owner_sensor";
+    public static SensorType<CatOwnerSensor> OWNER;
```

Comment:
```suggestion
    public static SensorType<CatOwnerSensor> OWNER;

```
Empty line I think just looks nice lol

Reply:
Oh yea, I missed it. Thanks